helpdesk_mgmt/controllers/main.py

```diff
@@ -90,15 +98,18 @@ def submit_ticket(self, **kw):
         new_ticket = request.env["helpdesk.ticket"].sudo().create(vals)
         new_ticket.message_subscribe(partner_ids=request.env.user.partner_id.ids)
         if kw.get("attachment"):
+            IrAttachment = request.env["ir.attachment"]
+            attachment_ids = IrAttachment
```

Comment:
what's the point of this?
```suggestion
            attachment_ids = request.env["ir.attachment"]
```

Reply:
`IrAttachment` is used in the create statement below, that's why the model is set once outside the loop